### ISSUE TITLE
Kali Linux Light. No AP

### ISSUE THREAD

**Issue by TaurosOX** [FluxionNetwork/fluxion#429]
Any help please. I got kali light (full kali linux dont work with my tc1100 laptop). Fluxion cant see any AP. Step 1 i choose language. Step 2 it says "Select a wireless attack for the access point
    ESSID: "[N/A]"  /  [N/A]
Channel:   [N/A]
    BSSID:   [N/A] ( [N/A] )
[1] Captive portal etc..
[2] Handshake Snopper etc...
[3] Back

**Comment by lipitb** [comment]
Yes, In my pc i have it too

**Comment by microhasher** [comment]
I use the Alpha AWUS1900 and Fluxion 4 gets stuck at the Start AP Service

**Comment by underscrew** [comment]
Well, something went very wrong in the last commits. Getting the same thing here.

**Comment by makmak05** [comment]
the same us well too. but its working .in my own experience first i choose handshake snooper then select the target when its done .you can  see your own AP ..the only problem is that you cant change another target victim you need to restart your wireless card.

**Comment by deltaxflux** [comment]
First buy a compatible wireless card.

**Comment by blueflame357** [comment]
i have found a solution to this problem.. my friend bought a 2nd hand awus1900 and it has a problem like yours.. but the new awus1900 that just came out of the box he bought yesterday has no issues.  this where i solved my friend's issue with the old awus1900..

when no AP's found
in the terminal

cd fluxion
airmon-ng check kill
 do not close the terminal.
then unplug the wireless adapter (not physically) in the settings USB uncheck it. wait for 10sec then plug in again USB check wireless driver.  wait 10 sec.. then back to the open terminal and 

./fluxion.sh 

attack again.
if still no AP's found, try again the step..

now everytime my friend uses his old awu1900, he do 

cd fluxion
airmon-ng check kill
unplug/plug wireles adapter
./fluxion.sh

works without a fail.. THIS SOLVES NOT ONLY AWUS1900.. BUT OTHER COMPATIBLE WIRELES ADAPTER THAT HAS PROBLEM SHOWING AP's. i tried with few....  i hope it helps you guys too.

and @makmak05  the latest fluxion asking you to change/select another target.. or same target answer with  (Y/n) it will "again" give you option to scan channels.

**Comment by rad4day** [comment]
tbh I didn't even notice the Issue described in the original post as an issue. As at this point in time there is no AP selected by the user the value is NA (Not availabe).

It's some weird UI, that's sure though. Maybe we should start a "small" UI Improvement project.

**Comment by blueflame357** [comment]
Yes i agree.. issue not properly decribed. Though On my understanding, i thought its about no AP scan result. As the other comment saying stucked at AP service.  well anyway whatever their issue is, they need to properly describe it in details to get appropriate help.. goodluck to them. 😅

**Comment by TaurosOX** [comment]
Well, i have no idea how to explain my issue. Check link please. There is a screenshot. 
Choosing any option return me to this menu again. 
https://drive.google.com/file/d/1eJAS6_aAAAomj7EZ00B0TBoAfQxmkMDZ/view?usp=drivesdk

**Comment by rad4day** [comment]
Just proceed, this is some weird UI thing fluxion does (at least thats what I think).
Maybe @strasharo or @deltaxflux  could clarify this for us?`=)

**Comment by rad4day** [comment]
I said proceed, not to abort or go back. Select your attack and run it.

**Comment by deltaxflux** [comment]
I have to rewrite the webinterface

**Comment by blueflame357** [comment]
the fluxion is perfect the way it is right now.... just sayin.. 
though most phone's not able to pop out the fake portal where they will need to put their pass key.
but if they are using pc or laptop. its perfect.

**Comment by rad4day** [comment]
@blueflame357 if it's an ssl error the same should happen on your computer otherwise you should really check your security settings and trusted CA....

The problem we're talking about here right now is the confusing UI. It really could need some sort of overhaul. I thaught it was broken, too. :')

**Comment by TaurosOX** [comment]
Rad4day, proceed  what? When i choose 1 or 2 nothing s change. I just see this pic all the time

**Comment by TaurosOX** [comment]
It's kali linux light 32 bit ( 4.14.0 2018-01-08). TP-LINK TL-WN310G. Tablet PC hp  TC1100

**Comment by rad4day** [comment]
@TaurosOX is your fluxion installation up to date?
Also [here is the link to the issue template](https://raw.githubusercontent.com/FluxionNetwork/fluxion/master/.github/ISSUE_TEMPLATE.md ) . Please copy this over and fill it out once so we get an overview about what is happening. You really shouldn't be stuck at this menu.

Also I'll clean this mess up and reopen this.

**Comment by MPX4132** [comment]
@deltaxflux I thought so too, but it seems that's not the case, unless I missed something.

**Comment by deltaxflux** [comment]
Ok I will close this issue until he answere.

**Comment by rad4day** [comment]
@deltaxflux @MPX4132 I flagged it as an bug specifically because of this comment

> Rad4day, proceed what? When i choose 1 or 2 nothing s change. I just see this pic all the time

Also I'd propose to change the "[N/A]" to something which is easier to understand when you use the script the first time like "Not selected yet" or disable the header until an access point was selected.

**Comment by deltaxflux** [comment]
Or maybe a short message run the handshake snooper attack first.

**Comment by MPX4132** [comment]
Yeah, hiding the header or showing a tip sound like a good idea. People don't even bother reading the wiki.

**Comment by TaurosOX** [comment]
Well, it seems recording video the easiest way to show u whats going on. Make a short video soon. Thank you for your assist

**Comment by MPX4132** [comment]
@TaurosOX Yeah, please do that, we'd appreciate it.

**Comment by TaurosOX** [comment]
Hello. Here is the record https://drive.google.com/file/d/1No6JyJmgx9NR_FKhWxmacbHzD5BimDQx/view?usp=drivesdk

**Comment by TaurosOX** [comment]
@MPX4132 Hope it helps

**Comment by TaurosOX** [comment]
I'm using a light version of Kali. May be this is the reason of this issue? Any help, mates

**Comment by MPX4132** [comment]
@TaurosOX Okay, by the looks of it, it seems the problem is NOT the IO library which is what takes user input and validates it. That means it could be a problem with the “sequencing framework” or another issue related to the attacks selected, I’ll look into it in a bit and let you know what to try to see if we can find the problem.

I think it’s an incompatibility issue between your bash and the sequencing framework.

Please run and post the output of the following:
```
ls -l $(which bash)
```

**Comment by rad4day** [comment]
@MPX4132  I don't know what you want to gain from the location/symlink of his bash file.
@TaurosOX  I would be interested in the output of the diagnostics script or at least the output of
 `bash --version`

**Comment by MPX4132** [comment]
@rad4day I want to know if it’s pointing to dash, rather than bash. As far as I’m concerned, dash has a few things stripped out and I doubt the sequencing framework would work on dash. Dash is typically bundled with “light” distros, so yeah.

**Comment by TaurosOX** [comment]
Here https://drive.google.com/file/d/1wGbq52hqw-LPQ0pow-BIXDOIP3YPxznH/view?usp=drivesdk

**Comment by rad4day** [comment]
Oh sorry, I'm pretty bad in reading my notifications as it seems.
Well your bash does not point at dash, which is good.

I'm sorry I'd like to verify if it is a problem with the (a little bit) outdated bash version, but I have almost no time right now.

@MPX4132 @deltaxflux  you probably know more about this then me

**Comment by TaurosOX** [comment]
@deltaxflux why did you close this issue? :(

**Comment by MPX4132** [comment]
@TaurosOX Can you pick up any networks with `airodump-ng`?
```
# Set interface to monitor mode
airmon-ng start wlanX # Replace wlanX with your interface identifier

airodump-ng wlanXmon
# Ctrl+c to stop capturing

# Once you're done, stop monitor mode with:
airmon-ng stop wlanXmon
```

**Comment by deltaxflux** [comment]
Sorry @TaurosOX was accidentally.

**Comment by MPX4132** [comment]
Hey @TaurosOX try launching fluxion in debug mode and post the output here:

```
./fluxion.sh --debug

# Once at the menu, launch the following and paste the output in your reply
tail -f /tmp/fluxion_debug_log 2> /dev/null
```

**Comment by TaurosOX** [comment]
Check screenshots that have attached
1. https://drive.google.com/file/d/1pSKrumVsgJC1YAtbcY7XkiGLLLhkKoxY/view?usp=drivesdk
2. https://drive.google.com/file/d/1wS5tZ160ltcl9_9gN5DzVW4kc08-NkCv/view?usp=drivesdk
3. https://drive.google.com/file/d/19vikC6UpFldgDsN6H5cR8nxuj1LoMA-1/view?usp=drivesdk

**Comment by MPX4132** [comment]
@TaurosOX You need to open two terminal windows, one where you launch fluxion and wait till the first menu shows up. Then run the second command I posted on the second window after you're at the menu in the first window (launch the `tail` command on the second window while the first window is on the menu).

Then try to go into one of the menus. Terminate fluxion with ctrl+c (the first window), and copy the contents of the second window and post them here as your reply.

**Comment by sonfiree** [comment]
Debug http://prntscr.com/k4w053

**Comment by MPX4132** [comment]
@sonfiree That wasn't a problem, unless you didn't properly configure your keys.

**Comment by TaurosOX** [comment]
Thank you for everyone! The issue has gone after toodays update. It works finally!

**Comment by fcmedo** [comment]
guys please me I have probelm 
fluxion scan doesn't search targit network

**Comment by fcmedo** [comment]
look ....[1] Captive Portal Creates an "evil twin" access point.
[2] Handshake Snooper Acquires WPA/WPA2 encryption hashes.
[3] Back

I chosse 2

[1] wlan0 [+]
[2] wlan1 [+]
[3] Repeat
[4] Back

I choose 1

[1] All channels (2.4GHz)
[2] All channels (5GHz)
[3] All channels (2.4GHz & 5Ghz)
[4] Specific channel(s)
[5] Back

I choose 1

after that he opens with me terminal but doesn't give me ap wifi network
just in show me BSSD.........

**Comment by fcmedo** [comment]
Sory I choose wlan0

**Comment by rad4day** [comment]
Digging up an old (probably unrelated) issue and commenting on that notifies anyone who has written anything in that issue... Which is not nice. You may open a new issue referencing this one if you think your problem is related.
